src/esm_runscripts/filedicts.py

```diff
@@ -23,17 +26,37 @@ class SimulationFile(dict):
 
     And, assuming config is as described above::
 
-        >>> sim_file = SimulationFile(config['echam']['files']['jan_surf'])
+        >>> sim_file = SimulationFile(config, ['echam']['files']['jan_surf'])
 
     You could then copy the file to the experiment folder::
 
         >>> sim_file.cp_to_exp_tree()
     """
 
-    # Do NOT implment an __init__ unless you really think you need to. dict
-    # takes care of this in a way is that is considerably smarter than any of us are.
-    #
-    # Please delete my annoying comments before merging into actual release ;-)
+    def __init__(self, full_config, attrs_address):
+        """
+        Parameters
+        ----------
+        full_config : dict
+            The full simulation configuration
+        attrs_address : str
+            The address of this specific file in the full config, separated by dots.
+        """
+        super().__init__()
+        attrs_dict = dpath.util.get(full_config, attrs_address, separator=".")
+        self.update(attrs_dict)
```

Comment:
```suggestion
        attrs_dict = dpath.util.get(full_config, attrs_address, separator=".")
        super().__init__(attrs_dict)
```
I think this will work and we get rid of the update